vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -30,8 +32,29 @@ using namespace facebook::react;
 
 namespace Microsoft::ReactNative {
 
-static YogaNodePtr make_yoga_node(YGConfigRef config) {
-  YogaNodePtr result(YGNodeNewWithConfig(config));
+static YGConfigRef make_yoga_config(winrt::Microsoft::ReactNative::ReactContext const &context, YGLogger logger) {
+  const auto config = YGConfigNew();
+  if (React::implementation::QuirkSettings::GetUseWebFlexBasisBehavior(context.Properties()))
+    YGConfigSetExperimentalFeatureEnabled(config, YGExperimentalFeatureWebFlexBasis, true);
+  auto errata = YGErrataAll;
+  if (!React::implementation::QuirkSettings::GetMatchAndroidAndIOSStretchBehavior(context.Properties()))
```

Comment:
My memory is that @acoates-ms added this when we discovered we were not setting `UseLegacyStretchBehaviour` in case anyone might have been relying on it, but I have doubts that anyone is actually using it.

If we are adding a more generic way to control per subtree, we may want to consider removing the instance-level QuirkSetting.

Reply:
I think the quirk setting with also just no-op on Fabric.

Reply:
It's not a huge problem. The ReactRootView will pull it's value off QuirkSettings, and to accommodate for this, I pull the "fallback" / default Errata value off the ReactRootView for a given node. So, we respect the instance settings here.

But you're right, I don't think we're using this at all for Fabric.